docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -12,15 +12,12 @@ const {
 	transformApiModel,
```

Comment:
Apparently prettier has not been running on this code. Many of the changes here are just prettier formatting changes. But since I think we want them, I'll leave them.

Reply:
It doesn't look like we set the docs dir up with prettier at all. I don't recall why, if there was an explicit reason. I do recall trying to add it to the directory prior to our repo-wide effort, but prettier chokes on Hugo's templates. When we do add it, we'll have to omit the website contents themselves from being formatted by it :/